astropy/modeling/rotations.py

```diff
@@ -27,11 +27,93 @@
 from .parameters import Parameter
 
 
-__all__ = ['RotateCelestial2Native', 'RotateNative2Celestial', 'Rotation2D']
+__all__ = ['RotateCelestial2Native', 'RotateNative2Celestial', 'Rotation2D',
+           'EulerAngleRotation']
 
 
 class EulerAngleRotation(Model):
     """
+    Implements Euler angle rotation.
+
+    Parameters
+    ----------
+    phi, theta, psi : float
+        Euler angles in deg
+    order : str
+        A 3 character string, a combination of 'x', 'y' and 'z',
+        where each character denotes an axis in 3D space.
+    """
+
+    inputs = ('alpha', 'delta')
+    outputs = ('alpha', 'delta')
+
+    phi = Parameter(default=0, getter=np.rad2deg, setter=np.deg2rad)
+    theta = Parameter(default=0, getter=np.rad2deg, setter=np.deg2rad)
+    psi = Parameter(default=0, getter=np.rad2deg, setter=np.deg2rad)
+
```

Comment:
Maybe have a class attribute called `axes = ('x', 'y', 'z')`.  Maybe YAGNI, but seems like the sort of thing I like to hard-code as an attribute somewhere.